source/_patterns/04-pages/msa.mustache

```diff
@@ -24,21 +24,25 @@
       </div>
 
 
-    <div class="wrapper">
+    <div class="wrapper wrapper--listing wrapper--listing-selector">
 
       <div class="grid">
 
         <div class="listing-column-container">
 
-          <div class="grid__item large--eight-twelfths listing-column-main">
+          {{#viewSelector}}
+            {{> molecules-view-selector }}
+          {{/viewSelector}}
+
+          <div class="grid__item large--eight-twelfths listing-column-main primary-column" id="primaryListing">
```

Comment:
The "one-whole" class is needed here too, as in home.mustache.

Reply:
Done, also "large--eight-twelfths" is changed to "x-large--eight-twelfths"